web/api/web_api_v1.c

```diff
@@ -1462,6 +1462,64 @@ int web_client_api_request_v1_dbengine_stats(RRDHOST *host __maybe_unused, struc
 }
 #endif
 
+#ifdef NETDATA_DEV_MODE
+int web_client_api_request_v1_logs(RRDHOST *host __maybe_unused, struct web_client *w, char *url __maybe_unused) {
+    if (!netdata_ready)
+        return HTTP_RESP_BACKEND_FETCH_FAILED;
+
+    const char *kind = NULL;
+
+    while (url) {
+        char *value = strsep_skip_consecutive_separators(&url, "&");
+        if (!value || !*value)
+            continue;
+
+        char *name = strsep_skip_consecutive_separators(&value, "=");
+        if(!name || !*name)
+            continue;
+
+        if (!name || !*name || !value || !*value)
+            return HTTP_RESP_NOT_FOUND;
+
+
+        if (!strncmp(name, "kind", strlen("kind"))) {
+            kind = value;
+        }
+    }
+
+    if (!kind)
+        return HTTP_RESP_NOT_FOUND;
+
+    const char *filename = NULL;
+
+    if (!strcmp(kind, "error"))
+        filename = stderr_filename;
+    else if (!strcmp(kind, "access"))
+        filename = stdaccess_filename;
+    else if (!strcmp(kind, "collector"))
+        filename = stdcollector_filename;
+    else if (!strcmp(kind, "health"))
+        filename = stdhealth_filename;
```

Comment:
I agree that `debug` and `aclk-ssl` are logs that are used more for developers, but I would suggest to have them inside `NETDATA_DEV_MODE`.

Reply:
Which variables point to the paths of `debug` and `aclk-ssl`?

Reply:
For `aclk`  we have the [aclklog](https://github.com/netdata/netdata/blob/6bba90a5320770220bb44acce4ca97e1604aefce/libnetdata/log/log.c#L38-L40).
Now for `debug`, we are using another pattern, so we can ignore it for while.